04 Strategy Library/21 Momentum Effect in Stocks/02 Method.html

```diff
@@ -6,32 +6,15 @@ <h4>Momentum Calculation</h4>
-            item = IndicatorDataPoint(self.symbol, tuple.Index, float(tuple.close))
-            self.MOM.Update(item)
-  </pre>
-  </div>
 <p>
-  Dictionary <code>self.symbolDataDict</code> is used to save the momentum class instance <code>SymbolData</code> for each symbol.
-  In <code>OnSecuritiesChanged</code> event method, we add the newly selected symbol to the dictionary and initialize the momentum indicator with the history request. For symbols removed from the universe, we remove it from the dictionary. Each day in <code>OnData</code>, the 
-  Momentum indicator for all symbols in the dictionary will be updated with the latest closing price.
+  Dictionary <code>self.mom</code> is used to save the LEAN Momentum class instance <code>Momentum</code> for each symbol.
+  In <code>OnSecuritiesChanged</code> event method, we add the newly selected symbol to the dictionary and initialize the momentum indicator with the history request. For symbols removed from the universe, we remove it from the dictionary and liquidate its positions. Each day in <code>OnData</code>, the Momentum indicator for all symbols in the dictionary will be updated with the latest closing price.
 </p>
 <p>
   We choose a period of 12 months for the momentum indicator. Stocks with the best 12-month momentum (12-month performance) are then added to our portfolio and are weighted equally.
 </p>
 
 <h4> Monthly Rebalance</h4>
 <p>
-  The portfolio is rebalanced once a month. The coarse and fine universe selection is set to default to run at midnight once a day. To make the universe selection run at the first trading day each month, we use the bool variable
-  <code>self.monthly_rebalance</code> to manage the universe selection. At the start of each month, the universe selection will filter new stocks. On all other days, the universe selection function will return the same symbols. In contrast to returning an empty list, returning the same symbols as before is a better way for monthly rebalance universe selection. Since if there are no open positions for certain symbol, returning empty list will stop the data subscription of that symbol halt updates of the indicator.
+  The portfolio is rebalanced once a month. The coarse and fine universe selection is set to default to run at midnight once a day. To make the universe selection run at the first trading day each month, we use the int variable <code>self.month</code> that tracks the current month to manage the universe selection. At the start of each month, the universe selection will filter new stocks. On all other days, the universe selection function will return the same symbols <code>self.symbols</code>. In contrast to returning an empty list, returning the same symbols as before is a better way for monthly rebalance universe selection. Since if there are no open positions for certain symbol, returning empty list will stop the data subscription of that symbol halt updates of the indicator.
```

Comment:
"Since if there are no open positions for certain symbol, returning empty list will stop the data subscription of that symbol halt updates of the indicator.
```suggestion
 If there are no open positions for certain symbol, returning an empty list will stop the data subscription of that symbol halt updates of the indicator.
```